input/data/2006/2006ApJ...639..761A/tev-000138.yaml

```diff
@@ -0,0 +1,19 @@
+source_id: 138
+reference_id: 2006ApJ...639..761A
+telescope: magic
+data_entry_status: incomplete
+
+data:
+  livetime: 6h # The paper says '~6hours'
+  significance: 8.2
+  excess: 246
+
```

Comment:
I think, no morph-type and no position is given in the paper, is it?

Reply:
This is an AGN. Yes, no position or morhphology was measured.